src/context/WalletProvider/KeepKey/setupKeepKeySdk.ts

```diff
@@ -0,0 +1,20 @@
+import { KeepKeySdk } from '@keepkey/keepkey-sdk'
+
+export const setupKeepKeySDK = async () => {
+  let serviceKey = window.localStorage.getItem('@app/serviceKey')
+  let config: any = {
+    apiKey: serviceKey,
+    pairingInfo: {
+      name: 'ShapeShift',
+      imageUrl: 'https://assets.coincap.io/assets/icons/fox@2x.png',
```

Comment:
grab this from the asset service as we have a single source of truth. we have a `foxAssetId` constant floating around you can use. you can use the asset service singleton directly rather than a selector.

Reply:
We need a static URL hosting the Dapps icon. Something I learned from coincap days was the constant roar of coins and app constantly updating their logos. This imgUrl is stored in our dapp database and is equivalent the pairing icon in wallet-connect. by using static urls we allows dapps to update their logos at will. I poked shapeshift.com looking for a static asset url for the fox logo but didn't find a good solution. I welcome any suggestions or a static url to reference here.

Reply:
without checking, `getAssetService().getAll()[foxAssetId]?.icon`

Reply:
we need a static URL hosting the icon. an example of a static url would be ```https://shapeshift.com/foxIcon.png```

Reply:
yep. i know. what i gave you will return a string, containing a static url, hosting the icon.